x-pack/plugins/security_solution/cypress/e2e/detection_response/rule_management/authorization/all_rules_read_only.cy.ts

```diff
@@ -26,7 +28,7 @@ import { SECURITY_DETECTIONS_RULES_URL } from '../../../../urls/navigation';
 
 const MISSING_PRIVILEGES_CALLOUT = 'missing-user-privileges';
 
-describe('All rules - read only', () => {
+describe('All rules - read only', { tags: tag.ESS }, () => {
```

Comment:
I'm not sure that this is just ESS. Regardless of how there's different roles, the behavior with read only should be the same in both cases I think?

Reply:
I tagged it as just ESS since per my understanding roles and the setup of them are going to be slightly different on serverless.

Reply:
We can revisit this later when we figure out how to test against different roles in Serverless.